src/themes/elife/styles.css

```diff
@@ -123,6 +125,17 @@ main {
   }
 }
 
+:--datePublished {
+  color: var(--color-text-secondary);
+  display: block;
+  font-family: 'Noto Sans', Arial, Helvetica, sans-serif;
+  font-size: calc(var(--text-size-rem-small) * 1rem);
+  letter-spacing: var(--letter-spacing);
+  line-height: calc(var(--baseline-measure-rem) / var(--text-size-rem-small));
```

Comment:
This is not a move of the code from https://github.com/stencila/thema/pull/142/files#diff-44d5a42c7d7f66640339dcf314193b4cL151, this line is different (& out of date).